homeassistant/components/gardena_bluetooth/sensor.py

```diff
@@ -51,6 +52,41 @@ class GardenaBluetoothSensorEntityDescription(SensorEntityDescription):
         native_unit_of_measurement=PERCENTAGE,
         char=Battery.battery_level,
     ),
+    GardenaBluetoothSensorEntityDescription(
+        key=Sensor.battery_level.uuid,
+        translation_key="sensor_battery_level",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.BATTERY,
+        entity_category=EntityCategory.DIAGNOSTIC,
+        native_unit_of_measurement=PERCENTAGE,
+        char=Sensor.battery_level,
+        connected_state=Sensor.connected_state,
+    ),
+    GardenaBluetoothSensorEntityDescription(
+        key=Sensor.value.uuid,
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.MOISTURE,
+        native_unit_of_measurement=PERCENTAGE,
+        char=Sensor.value,
+        connected_state=Sensor.connected_state,
+    ),
+    GardenaBluetoothSensorEntityDescription(
+        key=Sensor.type.uuid,
+        translation_key="sensor_type",
+        entity_category=EntityCategory.DIAGNOSTIC,
+        char=Sensor.type,
+        connected_state=Sensor.connected_state,
+    ),
+    GardenaBluetoothSensorEntityDescription(
+        key=Sensor.measurement_timestamp.uuid,
+        translation_key="sensor_measurement_timestamp",
+        state_class=SensorStateClass.MEASUREMENT,
+        device_class=SensorDeviceClass.DATE,
```

Comment:
Shouldn't this be TIMESTAMP? as the name implies that

Reply:
I think it's a full date and time. Not just a time. Will see when i get a user to test.

Reply:
Oh I'll need to recheck the docs on when to use what, it could be that this is right, but I'm not sure then

Reply:
Docs are not very clear now that i look :). But since this will be a datetime object, it probably should be timestamp.